### DISCUSSION TITLE
plotting config  threadripper pro 3995wx 64/128

### DISCUSSION THREAD

**Discussion by newland33** [swar/Swar-Chia-Plot-Manager#772] [Q&A]
i want to start with a  second plotter and harvester. what settings can be done . i want to share with plotterfriends timings, speeds etc :)
or u have settings to start for me
cpu threadripper pro 3995wx 64/128
ram 128gb ddr4 3200 ecc
3x 4TB wd an1500 nvme 
3x2 TB samsung 980 pro

3x 14TB hhd in computer
10x36 TB RAID 0 USB3 hhds

**Comment by NEYZERO** [comment]
i need too
